README.md

```diff
@@ -15,6 +15,7 @@ We provide 4 sets of captures; for each subject:
 And code to train personalized decoders:
 * Relightable Gaussian Codec Avatar heads,
 * Relightable hands
+* Universal relightable hands
```

Comment:
nit: would you mind making these lines clickable so they go to the paper or arxiv or project page?

Reply:
arXiv link added